mscorlib/system/globalization/charunicodeinfo.cs

```diff
@@ -322,7 +338,7 @@ internal static bool IsWhiteSpace(char c)
             }
             return (((double*)s_pNumericValues)[pBytePtr[(ch & 0x000f)]]);
 #else
-            return (((double*)s_pNumericValues)[pBytePtr[(ch & 0x000f)]]);
+            return EndianSwap(((double*)s_pNumericValues)[pBytePtr[(ch & 0x000f)]]);
```

Comment:
@marek-safar Also, note that there are several EndianSwap() methods already in that file as there was an earlier fix to this file relating to endian issues. It was in commit 41a6d3d6a5432b964b6f8439b6375ffad6f2e6cb. I assume these  invocations could also be changed to the Mono.Security.BitConverterLE.xxxx methods.